test/system/032-sig-proxy.bats

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bats
+
+load helpers
+
+@test "podman sigkill" {
+    $PODMAN run -i --name foo $IMAGE sh -c 'trap "echo BYE;exit 0" INT;echo READY;while :;do sleep 0.1;done' &
+    local kidpid=$!
+
+    # Wait for container to appear
```

Comment:
I don't think this loop is necessary.  The container should be there instantly if you run it with a -d flag.